src/entities/mellow-stateless/config/config.ts

```diff
@@ -0,0 +1,337 @@
+          {
+            address: '0x5e25887EA7507e1646C3637c5CCCE29D7D2c1E9C',
+            weight: 0,
+            pools: ['Compound - USDC', 'Compound - USDC'],
+            withdrawable: true,
+            estimatedHistoricApy: [10, 10],
+          },
+          {
+            address: '0x5e25887EA7507e1646C3637c5CCCE29D7D2c1E9C',
+            weight: 0,
+            pools: ['Compound - USDC', 'Compound - USDC'],
+            estimatedHistoricApy: [30, 30],
+            withdrawable: true,
+          },
+          {
+            address: '0xAb0f8CeE5fa7e3D577d1a546Aeb11fE5c768c75E',
+            weight: 0,
+            pools: ['Compound - USDC'],
+            estimatedHistoricApy: [30, 30],
+            withdrawable: true,
+          },
+          {
+            address: '0x4972C5f24E6EDfD479ba989b204bD376503D48d8',
+            weight: 100,
+            pools: ['Compound - USDC'],
+            estimatedHistoricApy: [30, 30],
+            withdrawable: false,
+          },
+        ],
+      },
+      {
+        optimiser: '0xB1EBCC0367c775D9447DFb0B55082aA694113ae0',
+        isVault: false,
+        title: 'MELLOW - USDT',
+        description:
+          'The Mellow LP Optimiser runs a permissionless strategy that takes deposits and generates optimised LP fees by providing liquidity on Voltz Protocol.',
+        show: true,
+        soon: false,
+        deprecated: false,
+        vaults: [
+          {
+            address: '0x6D2b3Cab0dD3367862C3Be90E702D976102482b3',
+            weight: 0,
+            pools: ['Compound - USDT'],
+            estimatedHistoricApy: [15, 15],
+            withdrawable: true,
+          },
+          {
+            address: '0x27F41ce17725B79cfD195475F3e2B3231a27640e',
+            weight: 100,
+            pools: ['Compound - USDT'],
+            estimatedHistoricApy: [15, 15],
+            withdrawable: false,
+          },
+        ],
+      },
+      // {
+      //   optimiser: '0x7AaA278531D0baCb2aC483be3edDFf83E09564Aa',
+      //   isVault: false,
+      //   title: 'Cost Test MELLOW - ETH',
+      //   description:
+      //     'The Mellow LP Optimiser runs a permissionless strategy that takes deposits and generates optimised LP fees by providing liquidity on Voltz Protocol.',
+      //   show: true,
+      //   soon: false,
+      //   deprecated: false,
+      //   vaults: [
+      //     {
+      //       address: '0x4FE3444AC2Ee16cAF4661fba06186b09E4F0a706',
+      //       weight: 50,
+      //       pools: ['Compound - ETH'],
+      //       estimatedHistoricApy: [10, 10],
+      //       withdrawable: true,
+      //     },
+      //     {
+      //       address: '0x5de7a5BbEDcE4a739b8a8D1cdA15D71924BDC9f7',
+      //       weight: 50,
+      //       pools: ['Compound - ETH'],
+      //       estimatedHistoricApy: [10, 10],
+      //       withdrawable: true,
+      //     },
+      //   ],
+      // },
+    ],
+  },
+};
+
+let cachedConfig: NetworkConfiguration | null = null;
```

Comment:
why do we need to keep this in a cache? seems easy just to ask for networkConfigurations[network]

Reply:
You can initialize it once and then use, without passing network props everywhere. It'd be great if we can do this with all variables that don't change during sessions (e.g. provider, configurations etc.)

Reply:
so when we introduce a listener for wallet network changes, the UI must instantiate the SDK again, right?

Reply:
hmm... we can't do this at this moment because each UI deployment is assigned to a specific network (the env vars cannot be changed during sessions).